README.md

```diff
@@ -1,6 +1,6 @@
 **cocotb** is a coroutine based cosimulation library for writing VHDL and Verilog testbenches in Python.
 
-[![Documentation Status](https://readthedocs.org/projects/cocotb/badge/?version=latest)](https://docs.cocotb.org/en/latest/)
+[![Documentation Status](https://readthedocs.org/projects/cocotb/badge/?version=stable)](https://docs.cocotb.org/en/stable/)
```

Comment:
I'd argue this badge should continue to track latest, to be aligned with the build status badge.